gcloud/pubsub/message.py

```diff
@@ -15,6 +15,11 @@
 """Define API Topics."""
 
 import base64
+import datetime
+
+import pytz
+
+RFC3369 = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Yes `gcloud._helpers` is what I had in mind.

Also keep in mind RFC3339 covers many more than this template, so it's not entirely "accurate" to call it RCF3339.